generated/nidaqmx/_install_daqmx.py

```diff
@@ -0,0 +1,154 @@
+import urllib.request
```

Comment:
This seems to be the same file as the one is src/handwritten.

Reply:
The codegen process copies the handwritten files to generated/nidaqmx.

Reply:
I have removed in the latest changes.